sdk/storage/test/datalake/path_client_test.hpp

```diff
@@ -7,12 +7,14 @@
 
 namespace Azure { namespace Storage { namespace Test {
 
-  class PathClientTest : public FileSystemClientTest {
+  class DataLakePathClientTest : public DataLakeFileSystemClientTest {
   protected:
     static void SetUpTestSuite();
     static void TearDownTestSuite();
 
-    static std::shared_ptr<DataLake::PathClient> m_pathClient;
+    static std::vector<Files::DataLake::Acl> GetInterestingAcls();
```

Comment:
what's so interesting?

Reply:
The fact that it works :).

Reply:
it's confusing.

Reply:
we should give it a more descriptive name

Reply:
Changed to `GetValidAcls`